main/glossary/README.md

```diff
@@ -67,151 +104,379 @@ synchronously compared for structural equivalence.
 
 ## Seat
 
-Zoe uses seats to represent offers, and has two seat [facets](#facet)  a ZCFSeat and a UserSeat.
+Zoe uses seats to represent [offers](#offer), and has two seat [facets](#facet); a `ZCFSeat` and a `UserSeat`.
+The term comes from the expression "having a seat at the table" with regards to participating in a
+negotiation.
 
 Seats represent active offers and let contracts and users interact with them. ZCFSeats are used 
 within contracts and with `zcf.` methods. User seats represent offers external to Zoe and the 
 contract. The party who exercises an invitation and sends the `offer()` message to Zoe 
 gets a UserSeat that can check payouts' status or retrieve their results.
 
+For more details, see the [ZCFSeat documentation](/zoe/api/zoe-contract-facet.md#zcfseat-object) and 
+the [UserSeat documentation](/zoe/api/zoe.md#userseat-object).
+
+## SeatStagings
+`seatStagings` are associations of seats with reallocations.
+
+## SES (Secure ECMAScript)
+
+SES is a standards-track extension to the JavaScript standard. It
+provides a secure platform for executing programs. With SES, you can run code you don't
+completely trust, without being vulnerable to bugs or bad intentions. 
+See the [SES section of the Distributed JavaScript Programming Guide](/distributed-programming.md#secure-ecmascript-ses) for 
+more details. 
+
+## Simoleons
+An imaginary currency Agoric documentation uses in examples.
+
+## Terms
+Contract instances have associated terms, gotten via `E(zoe).getTerms(instance)`,
+which include the instance's associated [issuers](#issuer), [brands](#brand), and any custom terms. For 
+example, you might have a general auction contract. When someone instantiates it,
+they provide terms applicable only to that instance. For example, for some instances of 
+the auction, they want the minimum bid set at $1000. At other instances, they'd like
+it set at $10. They can specify the instance's minimum bid amount in its terms.
+
 ## Value
 
 Values are the part of an [amount](#amounts) that describe the value of something
 that can be owned or shared: How much, how many, or a description of a unique asset, such as
-Pixel(3,2), $3, or ‘Right to occupy on Tuesdays’. [Fungible](#fungible) values are usually 
+Pixel(3,2), $3, or 'Seat J12 for the show September 27th at 9:00pm'.
+[Fungible](#fungible) values are usually 
 represented by natural numbers. Other values may be represented as strings naming a particular
 right, or an arbitrary object representing the rights at issue. The latter two examples 
 are usually [non-fungible](#non-fungible) assets. Values must be [Comparable](#comparable).
 
-See the [ERTP Guide's Value section](/ertp/guide/amounts.md#values) for more information.
+For more information, see the [ERTP Guide's Value section](/ertp/guide/amounts.md#values).
+
+## Vat 
+A vat is a unit of isolation. To paraphrase the Las Vegas advertising slogan, what happens in a vat stays in that vat.
```

Comment:
I like the snap of this, but I'm not sure it's helpful. What happens in a vat can drastically affect objects in other vats.